devices/generic/subdevices/color_light.json

```diff
@@ -0,0 +1,36 @@
+{
+    "schema": "subdevice1.schema.json",
+    "type": "$TYPE_COLOR_LIGHT",
+    "name": "Color light",
+    "restapi": "/lights",
+    "order": 10,
+    "uuid": ["$address.ext", "0x01"],
+    "items": [
+        "cap/color/xy/blue_x",
+        "cap/color/xy/blue_y",
+        "cap/color/xy/green_x",
+        "cap/color/xy/green_y",
+        "cap/color/xy/red_x",
+        "cap/color/xy/red_y",
+        "state/alert",
+        "state/bri",
+        "state/colormode",
+        "state/on",
+        "state/reachable",
+        "state/x",
```

Comment:
I do think we might have a couple of lights that don't support all colour capabilities.  Best make the following items optional:
- `state/x`, `state/y` are mandatory only when `cap/color/capabilities` includes `xy`;
- `state/hue`, `state/sat` are mandatory only when `cap/color/capabilities` include `hs`;
- `state/effect` is mandatory only when `cap/color/capabilities` includes `effect`.

Note that either `state/x` and `state/y` or `state/hue` and `state/sat` are mandatory (or the light would be a _Dimmable Light_), but that might be too complex for validator.

Reply:
This case is now handle from the validator rule